app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  include UsersHelper
+
+  def new; end
+
+  def create
+    # @user = User.new(user_params)
+    @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
Do not use mass assignment, define a private method, user_params, where you whitelist the attribute you pass to your @user variable. Refer to Micheal Hartl, Chapter 7, for reference.